Userland/Libraries/LibGfx/GradientPainting.cpp

```diff
@@ -55,6 +55,8 @@ class GradientLine {
     GradientLine(int gradient_length, Span<ColorStop const> color_stops, Optional<float> repeat_length, UsePremultipliedAlpha use_premultiplied_alpha = UsePremultipliedAlpha::Yes)
         : m_repeating { repeat_length.has_value() }
         , m_start_offset { round_to<int>((m_repeating ? color_stops.first().position : 0.0f) * gradient_length) }
+        , m_color_stops { color_stops }
+        , m_use_premultiplied_alpha { use_premultiplied_alpha }
```

Comment:
I don't know who merged this in the first place, but we never use `m_foo { ... }`, only `m_foo(...)`. (just thinking out loud, no change needed in this PR)

Reply:
Ah, that's a force of habit of mine... I'll make a note to fix that with my next gradient patch :sweat_smile: